package.json

```diff
@@ -89,12 +89,13 @@
     "lint_md": "mdlint $(git ls-files '*.md')",
     "mem_backend": "S3BACKEND=mem node index.js",
     "perf": "mocha tests/performance/s3standard.js",
-    "start": "npm-run-all --parallel start_dmd start_s3server",
+    "start": "npm-run-all --parallel start_dmd start_management start_s3server",
     "start_mongo": "npm run cloudserver",
     "start_mdserver": "node mdserver.js",
     "start_dataserver": "node dataserver.js",
     "start_s3server": "node index.js",
     "start_dmd": "npm-run-all --parallel start_mdserver start_dataserver",
+    "start_management": "node managementAgent.js",
```

Comment:
@ssalaues when this PR is merged (and a new docker image built) the cloudserver-manager pod will need to use this command line